### ISSUE TITLE
Replace chat with team toggle functionality

### ISSUE THREAD

**Issue by slipcor** [AllYourBlocks/ChunkWars#38]
Given that this is possible, which I did not check, would it be a good idea to implement a "toggle" for the chat system so people can "talk regularly" and that message only gets sent to their team and spectators?

People have complained about that, and by design spectators do not get the team messages.

A simple fix for that one thing could be adding spectators to both teams, too. Not sure about side effects of that though.

**Comment by ffaen** [comment]
You can only be on one team at once, so this isn't really possible I believe...

**Comment by slipcor** [comment]
Yep tested and verified. Players "can join" a second team but they get removed from the first in the process :(
